backend/infrahub/trigger/tasks.py

```diff
@@ -0,0 +1,84 @@
+
+if TYPE_CHECKING:
+    from uuid import UUID
+
+
+@task(name="trigger-setup", task_run_name="Setup triggers in task-manager")
+async def setup_triggers(client: PrefectClient) -> None:
+    log = get_run_logger()
+
+    # -------------------------------------------------------------
+    # Retrieve existing Deployments and Automation from the server
+    # -------------------------------------------------------------
+    deployment_names = list({name for trigger in triggers for name in trigger.get_deployment_names()})
+    deployments = {
+        item.name: item
+        for item in await client.read_deployments(
+            deployment_filter=DeploymentFilter(name=DeploymentFilterName(any_=deployment_names))
+        )
+    }
+    deployments_mapping: dict[str, UUID] = {name: item.id for name, item in deployments.items()}
+    existing_automations = {item.name: item for item in await client.read_automations()}
+
+    builtin_automations = [
+        item.name for item in await client.read_automations() if item.name.startswith(TriggerType.BUILTIN.value)
+    ]
+    trigger_names = [trigger.generate_name() for trigger in triggers]
+
+    _, to_delete, _ = compare_lists(list1=builtin_automations, list2=trigger_names)
+
+    # -------------------------------------------------------------
+    # Create or Update all builtin triggers
+    # -------------------------------------------------------------
+    for trigger in triggers:
+        automation = AutomationCore(
+            name=trigger.generate_name(),
+            description=trigger.description,
+            enabled=True,
+            trigger=trigger.trigger.get_prefect(),
+            actions=[action.get_prefect(mapping=deployments_mapping) for action in trigger.actions],
+        )
+
+        existing_automation = existing_automations.get(trigger.name, None)
+
+        if existing_automation:
+            await client.update_automation(automation_id=existing_automation.id, automation=automation)
+            log.info(f"{trigger.name} Updated")
+        else:
+            await client.create_automation(automation=automation)
+            log.info(f"{trigger.name} Created")
+
+    # -------------------------------------------------------------
+    # Delete Builtin Triggers that shouldn't be there
+    # -------------------------------------------------------------
+    for item_to_delete in to_delete:
+        existing_automation = existing_automations.get(item_to_delete, None)
+
+        if not existing_automation:
+            continue
+
+        await client.delete_automation(automation_id=existing_automation.id)
+        log.info(f"{item_to_delete} Deleted")
+
+    # -------------------------------------------------------------
+    # Delete Deprecated triggers
+    # -------------------------------------------------------------
+    for trigger_name in DEPRECATED_STATIC_TRIGGER_NAMES:
+        existing_automation = existing_automations.get(trigger_name, None)
```

Comment:
We can remove the `, None` from the .get() statements here and above as `None` would be returned regardless if the key doesn't exist.